Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -630,16 +630,16 @@ class TabTrayController: UIViewController {
     }
 
     private func openNewTab(request: NSURLRequest? = nil) {
-        toolbar.addTabButton.userInteractionEnabled = false
+        toolbar.userInteractionEnabled = false
 
         // We're only doing one update here, but using a batch update lets us delay selecting the tab
         // until after its insert animation finishes.
         self.collectionView.performBatchUpdates({ _ in
             let tab = self.tabManager.addTab(request, isPrivate: self.privateMode)
             self.tabManager.selectTab(tab)
         }, completion: { finished in
+            self.toolbar.userInteractionEnabled = true
             if finished {
```

Comment:
Doesn't look like this is quite the case. At least for the issue here. Not sure about the PrivateMode toggling. This seems to be related to the MenuPresentationAnimator which is responsible for transitioning the current view to the one with the menu.